package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "structurize",
-  "version": "1.3.13",
+  "version": "1.3.14",
```

Comment:
This change isn't backward compatible and as such should be a major version bump, not a patch version.